Felienne/hedy
# [CHORE] Switch from YAML to Babel

**Description**
In this PR we implement a huge chore where we switch from YAML to Babel (gettext) for or text translations. As the current YAML structure is implemented in all parts of our code base this is a very big operation. This is currently a draft as to keep track of the merging conflicts and testing issues.

**Fixes**
This PR fixes #2030.

**How to test**
As this is a (big) clean-up no change should be visible from a user perspective. However, it does need extensive testing to make really really sure we didn't break anything.

**Last things to do before merging**
- Copy `unsaved_changes` string to the `client_messages` YAMLs
- Fix that this is working (as it is currently broken)
- Verify that all pages and error messages are valid

**List of languages processed**
The structure is done, however all languages need to be manually copied  from the YAMLs to the .po files. In the list below we keep track of the languages in process (this is constantly updated):

- ~ar~
- ~bg~
- ~bn~
- ~cs~
- ~de~
- ~el~
- ~en~
- ~es~
- ~fr~
- ~fy~
- ~hi~
- ~hu~
- ~id~
- ~it~
- ~nl~
- ~pt_BR~
- ~pt_PT~
- ~sw~
- ~tr~
- ~zh_Hans~
